basculeLogging.go

```diff
@@ -53,6 +53,6 @@ func setLogger(logger *zap.Logger, lf ...LoggerFunc) func(delegate http.Handler)
 }
 
 func getLogger(ctx context.Context) *zap.Logger {
-	logger := sallust.Get(ctx).With(zap.Time("ts", time.Now().UTC()), zap.Any("caller", zap.WithCaller(true)))
+	logger := sallust.Get(ctx).With(zap.Time("ts", time.Now().UTC())).WithOptions(zap.WithCaller(true))
```

Comment:
l see you have a config field `disableCaller` but this is hardcode as true, did you mean to use the config field `disableCaller` as a var for `zap.WithCaller`?

Reply:
so when i add callerKey to the config - it automatically adds the caller field to all the logs even the ones before this logger with zap.WithCaller is set up. so I added disableCaller to keep it disabled at the beginning - and then used WithCaller(True) to add the caller field for these logs so that it would log the same information that scytale was logging before.